colorls.gemspec

```diff
@@ -30,21 +31,26 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/athityakumar/colorls'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
+  files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features|man)/})
   end
+  files << 'man/colorls.1'
+
+  spec.files         = files
   spec.bindir        = 'exe'
   spec.executables   = 'colorls'
   spec.require_paths = ['lib']
 
   spec.post_install_message = ColorLS::POST_INSTALL_MESSAGE
 
   spec.add_runtime_dependency 'filesize'
+  spec.add_runtime_dependency 'manpages'
   spec.add_runtime_dependency 'rainbow'
-  spec.add_runtime_dependency 'rake'
 
   spec.add_development_dependency 'bundler', '~> 1.15'
   spec.add_development_dependency 'diffy'
+  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'ronn'
```

Comment:
Should `ronn` rather be a runtime dependency? When a user does `gem install colorls`, is the manpages created by default by `ronn`? For that, we'd require `ronn` to be installed while installing colorls right?

Reply:
No, the man page is generated when building the package/gem, ie. when running rake.

Reply:
Hmm, so how does the user generate the man page? The user is going to only do gem install. Not git clone and rake install. Should we have a --setup flag for that?

Reply:
The manpage is part of the gem, it is included. This way it is only generated once and then uploaded to the gem server.

Reply:
If it's part of the gem, should it NOT be git-ignored? I thought that the user would be generating the man pages after installing colorls, that's why I asked to gitignore colorls.1 file. But if we're to generate the man file and push to Rubygems too, it shouldn't be gitignored.